README.md

```diff
@@ -27,11 +27,11 @@ Default: `undefined`
 
 Add a prefix to the file paths. Useful when your files are in a different URL than the page.
 
-#### options.basePath
+#### options.suffix
```

Comment:
There was some confusion around the `basePath` option. Depending on the outcome of #35 we'd better keep the documentation of that option in the Readme too for now.